src/apps/chifra/server/run.go

```diff
@@ -24,9 +23,6 @@ func RunServe(cmd *cobra.Command, args []string) error {
 	}
 
 	apiUrl := opts.Port
-	if !strings.Contains(apiUrl, "http") {
-		apiUrl = "http://localhost" + apiUrl
-	}
 
```

Comment:
You are right, I was rushing with it and got a bit confused. The issue was somewhere else, in Docker setup, so that's why I thought that this change fixes something :) We can ignore it or merge to get rid of printing things like `http://localhost9999`